nautobot_circuit_maintenance/tests/test_jobs.py

```diff
@@ -0,0 +1,69 @@
+"""Tests for Jobs being included."""
+# pylint: disable=missing-class-docstring,no-name-in-module
+from datetime import datetime
+from typing import NamedTuple
+
+from django.test import TestCase
+
+from nautobot_circuit_maintenance.jobs.site_search import check_for_overlap
+
+DATE_FORMAT = "%Y-%m-%d %H:%M:%S"
+
+
+class MockRecord(NamedTuple):
```

Comment:
why not import the `class Range(NamedTuple)` ?

Reply:
sorry Josh, not following your point. I don't see why the MockRecord is necessary, it's exactly the same (from my point of view)

Reply:
ping

Reply:
Conversed and I think we are good with the new name.